starknet_devnet/accounts.py

```diff
@@ -20,6 +21,8 @@ def __init__(self, starknet_wrapper):
         self.__n_accounts = starknet_wrapper.config.accounts
         self.__initial_balance = starknet_wrapper.config.initial_balance
 
```

Comment:
There is a weird empty line here.

Reply:
Addressed in https://github.com/Shard-Labs/starknet-devnet/pull/294/commits/e2b2e1ff522c91f9449b41c8209c652af6563155